clusters/build-clusters/01_cluster/sshd-bastion/coredns/coredns_config.yaml

```diff
@@ -8,11 +8,11 @@ data:
             rewrite stop name exact lnxocp01 sshd-0.bastion-z.svc.cluster.local
             rewrite stop name exact lnxocp02 sshd-1.bastion-z.svc.cluster.local
             rewrite stop name exact lnxocp03 sshd-2.bastion-z.svc.cluster.local
-            rewrite stop name regex ^.*libvirt-s390x-([0-9])-[0-9]\.$ sshd-{1}.bastion-z.svc.cluster.local
+            rewrite stop name regex ^.*libvirt-s390x-([0-9])-[0-9].* sshd-{1}.bastion-z.svc.cluster.local
```

Comment:
To test we need to update coredns changes in separate PR and restart the pods

Reply:
Move it here https://github.com/openshift/release/pull/20350. I'll update this PR when that one merges.